src/components/InstallTabs/__tests__/index.js

```diff
@@ -1,11 +1,10 @@
-/* eslint-disable @typescript-eslint/explicit-function-return-type */
 import React from 'react';
```

Comment:
I just noticed this is a .js file it should be a .tsx file like the other test files.

Reply:
Ah! Definitely @benhalverson. Will update this with `.tsx` and all other files related to the same issue. Thanks!